types/operator.go

```diff
@@ -23,13 +23,17 @@ type CommitteeMember struct {
 // HasQuorum returns true if at least 2f+1 items are present (cnt is the number of items). It assumes nothing about those items, not their type or structure
 // https://github.com/ConsenSys/qbft-formal-spec-and-verification/blob/main/dafny/spec/L1/node_auxiliary_functions.dfy#L259
 func (operator *Operator) HasQuorum(cnt int) bool {
-	return uint64(cnt) >= operator.Quorum
+	return uint64(cnt) >= operator.GetQuorum()
+}
+
+func (operator *Operator) GetQuorum() uint64 {
+	return 2*operator.FaultyNodes + 1
```

Comment:
Even though the formula is more general it depends on 2 variables instead of 1. Thus more chances of it somehow breaking...

Since we only support N=3F+1 atm, let's leave it